ctdcal/odf_io.py

```diff
@@ -164,7 +168,7 @@ def process_salts(ssscc_list, salt_dir=cfg.dirs["salt"]):
             )  # .round(4)
             _salt_exporter(saltDF, salt_dir)
 
-def printProgressBar(
+def print_progress_bar(
```

Comment:
Should probably switch to something like https://rich.readthedocs.io/en/stable/progress.html which supports jupyter notebooks as well.

Though this is only used in tests? Could probably be removed entirely if that is the case

Reply:
I used it on specific branches in the past. Assuming the `rich` import is quick, that would be a good replacement.

I think the progress bar can be removed from this commit.